tests/e2e/utils/utils.js

```diff
@@ -295,6 +295,38 @@ function extractWorkbenchData(data) {
   return workbenchData;
 }
 
+function getGrayLogSnapshotUrl(target_url, since_secs = 30) {
+  let monitoring_base_url;
+  let snapshot_url;
```

Comment:
Please use ``camelCase`` in JavaScript ;)

Reply:
this sounds like payback for my reviews in python :-D